### PR TITLE
Add "Find Kerberoastable Accounts" TTP

### PR DESCRIPTION
**Pull request by w0rk3r**

### CODE REVIEW

File: discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+id: 27b922c1-76ca-41c7-926f-1c36cabec122
+name: Find Kerberoastable Accounts
+description: Find Accounts that might be vulnerable to Kerberoasting
```
Comment (khyberspache) [inline discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml:3]: Add a bit more detail here for new users that will look at this TTP.  Provide more details on what you're looking for specifically with this TTP (SPNs that can be abused, etc).  This is too similar to the name

File: discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+tactic: discovery
+technique:
+  id: T1087.002
+  name: "Account Discovery: Domain Account"
+platforms:
+  windows:
+    cmd:
+      command: setspn.exe -Q */*
+      variants:
+        - command: setspn.exe -T {discovery.domain} -Q */*
```
Comment (khyberspache) [inline discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml:18]: we have a parser that can find domains. Let's make this chain together with other TTPs. You're submitted other TTPs that do domain discovery, so find the technique number for one of those and do `#{domain.T1XXX}`

Comment (khyberspache) [reply discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml:18]: also missing `#`

File: discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        $searcher.PropertiesToLoad.Add("pwdlastset")
+        $searcher.SearchScope = "Subtree"
+        $Results = $searcher.FindAll()
+        $Results | Where-Object {$_} | ForEach-Object {
+          $Properties = $_.Properties
+          $Properties.PropertyNames | ForEach-Object {            
+            $ObjectProperties[$_] = $Properties[$_][0]
+          }
+          New-Object -TypeName PSObject -Property $ObjectProperties
+        }
```
Comment (khyberspache) [inline discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml:38]: Have you tested this? Would the missing `;` cause any of these to no execute properly?

**Review by khyberspache — CHANGES_REQUESTED** [review]
Nice. Minor comments, will need a spin up a windows domain to test on

File: discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml (hunk 3 above)
Comment (w0rk3r) [reply discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml:38]: image

No problems here

File: discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml (hunk 2 above)
Comment (w0rk3r) [reply discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml:18]: Cool stuff

File: discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml (hunk 1 above)
Comment (w0rk3r) [reply discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml]: Added

**Comment by w0rk3r** [comment]
I think this one is ready to merge now @khyberspache

**Review by khyberspache — APPROVED** [review]

**Pull request merged**